lib/Github/HttpClient/CachedHttpClient.php

```diff
@@ -27,6 +27,13 @@ class CachedHttpClient extends HttpClient
     private $lastCachedResponse;
 
     /**
+     * $path + query parameter(s) if they exist.
+     *
+     * @var string
+     */
+    private $path;
```

Comment:
and btw, the naming is wrong: the value stored in this property is **not** the path